llvm/test/Transforms/NaryReassociate/preserving-debugloc-add-mul.ll

```diff
@@ -0,0 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=nary-reassociate -S | FileCheck %s
+
```

Comment:
```suggestion
; Test that NaryReassociate's tryReassociatedBinaryOp() propagates the
; debug location to new `add` and `mul` from the original binary operator
; they replaced (`%3` in both `@add_reassociate` and `@mul_reassociate`).
```
Can you please re-add your comment?

Reply:
Oh, I just forget that the `update_test_checks.py` would delete the comments.🥲